src/main/java/com/micetr0/controller/AccountController.java

```diff
@@ -0,0 +1,90 @@
+package com.micetr0.controller;
+
+import com.micetr0.model.Account;
+import com.micetr0.model.Composition;
+import javafx.util.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AccountController {
+
+    Account account = new Account();
+    List<Pair<String,String>> accounts = new ArrayList<>();
+
+    public void setModel(Account account) {
+        this.account = account;
+    }
+
+    private boolean checkCredentials(String username, String Password, List<Pair<String,String>> accounts){
+        boolean validAccount = false;
+
+        return validAccount;
+    }
+
+    public void initAccounts()
```

Comment:
Not needed, can remove, this will be created in test. Does not reflect the functionality of the system.